src/core/trulens/core/experimental/__init__.py

```diff
@@ -0,0 +1,441 @@
+from collections import defaultdict
+from enum import Enum
+import functools
+from types import FrameType
+from typing import (
+    Callable,
+    Dict,
+    Generic,
+    Iterable,
+    Mapping,
+    Optional,
+    Set,
+    TypeVar,
+    Union,
+)
+
+import pydantic
+from trulens.core.utils import python as python_utils
+from trulens.core.utils import text as text_utils
+
+T = TypeVar("T")
+
+
+class Feature(str, Enum):
+    """Experimental feature flags.
+
+    Use [TruSession.experimental_enable_feature][trulens.core.session.TruSession.experimental_enable_feature] to enable
+    these features.
+    """
+
+    OTEL_TRACING = "otel_tracing"
+    """OTEL-like tracing.
```

Comment:
please add a code example showing how to turn on otel_tracing

Reply:
Added example to Feature docstring.